cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -193,13 +194,23 @@ class AnalyzerCommand : OrtCommand(
             val repositoryPackageCurations = repositoryConfiguration.curations.packages
 
             if (ortConfig.enableRepositoryPackageCurations) {
-                add(SimplePackageCurationProvider(repositoryPackageCurations))
+                add("ort.yml" to SimplePackageCurationProvider(repositoryPackageCurations))
```

Comment:
If the check for duplicate ids is moved to `PackageCurationProvider` as suggested above here could be a check if the "ort.yml" id is already used.